From d06433ff1a1905436cfcde80e22ee51bd9591536 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 17 Sep 2024 09:59:31 -0700
Subject: [PATCH 2/2] ooo-file-unreg.c: Include poll.h instead of sys/poll.h

This fixes a warning e.g.

In file included from ooo-file-unreg.c:12:
/mnt/b/yoe/master/build/tmp/work/riscv32-yoe-linux-musl/liburing/2.7/recipe-sysroot/usr/include/sys/poll.h:1:2: warning: redirecting incorrect #include <sys/poll.h> to <poll.h> [-W#warnings]
    1 | #warning redirecting incorrect #include <sys/poll.h> to <poll.h>
      |  ^
1 warning generated.

Upstream-Status: Submitted [https://github.com/axboe/liburing/pull/1233]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 test/ooo-file-unreg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/ooo-file-unreg.c b/test/ooo-file-unreg.c
index d76e0fb..dd6ea55 100644
--- a/test/ooo-file-unreg.c
+++ b/test/ooo-file-unreg.c
@@ -9,7 +9,7 @@
 #include <sys/socket.h>
 #include <unistd.h>
 #include <stdlib.h>
-#include <sys/poll.h>
+#include <poll.h>
 
 #include "liburing.h"
 #include "helpers.h"
